spa/src/pages/ObjectProperties.vue

```diff
@@ -49,7 +49,7 @@
       <div class="grid" style="
         grid-template-columns: 450px auto;
       ">
-        <div id="thumbnail" class="justify-self-center"><img :src="imgFile" /></div>
+        <div id="thumbnail" class="justify-self-center"><img style="max-height: 450px; max-width: 450px;" :src="imgFile" /></div>

```

Comment:
I am not sure our version of tailwind supports this, but you can give arbitrary heights in classes now:
```suggestion
        <div id="thumbnail" class="justify-self-center"><img class="max-h-[450px] max-w-[450px]" :src="imgFile" /></div>
```